README.md

```diff
@@ -721,21 +730,17 @@ This will give you a local copy of the branch `daves`, and any update that has b
 <a name="i-want-to-create-a-new-remote-branch-from-current-local-one"></a>
 ### I want to create a new remote branch from current local one
 
-
 ```sh
-$ git config push.default upstream
-$ git push -u origin HEAD
+$ git push <remote>
 ```
 
-If you want to check out the other default configs which ```git push``` can take, visit the documentation for Git at https://git-scm.com/docs/git-config#git-config-pushdefault
-
-If you do not want to change the git configuration, you can also use:
+If you would also like to set that remote branch as upstream for the current one, use the following instead:
 
 ```sh
-$ git push -u <remote> HEAD
+$ git push -u <remote>
 ```
 
-With the ```upstream``` mode and the ```simple``` mode (default in Git 2.0), the following command will push the current branch w.r.t. the remote branch that has been registered previously with -u :
+With the `upstream` mode and the `simple` (default in Git 2.0) mode of the `push.default` config, the following command will push the current branch w.r.t. the remote branch that has been registered previously with -u :
```

Comment:
While we're here, change w.r.t. to 'with regards to', and put ticks around `-u`, and move the colon next to it?